Automattic/pocket-casts-ios
# Widgets: track install/uninstall

Add events to track when a widget is installed and uninstalled.

## To test

1. Ensure **tracksLogging** is enabled
2. Put the app in the background
3. Add some widgets
4. Open the app again
5. ✅ You should see `widget_installed` events fired with the correct `kind` and `family`
6. Put the app in the background again
7. Reopen
8. ✅ Nothing should be fired
9. Put the app in the background and add a new widget
10. ✅ An event tracking the installation of this widget should fire
11. Put the app in the background
12. Remove some widgets
13. ✅ You should see `widget_uninstalled` events fired with the correct `kind` and `family`
14. Put the app in the background
15. Add any of the removed widgets back
16. ✅ You should see `widget_installed` events fired with the correct `kind` and `family`

## Checklist
